### ISSUE TITLE
After updating the user profile, the new avatar is not displayed in the header, only after reloading the page (f5 in the browser)

### ISSUE THREAD

**Issue by n1k-crimea** [laravel/nova-issues#2912]
- Laravel Version: 8.6.0
- Nova Version: 3.10.0
- PHP Version: 7.4.10
- Database Driver & Version: MySQL 5.7
- Operating System and Version: Arch Linux
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

### Description:

After updating the user profile, the new avatar is not displayed in the header, only after reloading the page (f5 in the browser).
Did someone come across or might know how to fix it?

**Comment by davidhemphill** [comment]
Duplicate of #1708
